README.md

```diff
@@ -163,11 +163,11 @@ wheels for `torch`, `torchvision`, and `torch_xla` at
 
 | Version | GPU Wheel + Python 3.8 |
 | --- | ----------- |
+| 2.0 + CUDA 12.0 | `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/120/torch_xla-2.0-cp38-cp38-linux_x86_64.whl` |
 | 2.0 + CUDA 11.8 | `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/118/torch_xla-2.0-cp38-cp38-linux_x86_64.whl` |
-| 2.0 + CUDA 11.7 | `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/117/torch_xla-2.0-cp38-cp38-linux_x86_64.whl` |
 | 1.13 | `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/112/torch_xla-1.13-cp38-cp38-linux_x86_64.whl` |
-| nightly + CUDA 11.7 <= 2023/04/25| `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/117/torch_xla-nightly-cp38-cp38-linux_x86_64.whl` |
-| nightly + CUDA 11.7 >= 2023/04/25| `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/11.7/torch_xla-nightly-cp38-cp38-linux_x86_64.whl` |
+| nightly + CUDA 12.0 <= 2023/04/25| `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/120/torch_xla-nightly-cp38-cp38-linux_x86_64.whl` |
+| nightly + CUDA 12.0 >= 2023/04/25| `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.0/torch_xla-nightly-cp38-cp38-linux_x86_64.whl` |
```

Comment:
I don't remeber when is the first day we built 12.0, I don't think it is 04/25 through

Reply:
thanks for pointing out, we don't have wheel with CUDA12.0 before 2023/06/27, will update